### PR TITLE
adding BlackSamsung's coordinates

### PR DESCRIPTION
**Pull request by BlackSamsung**
@githubteacher What do ou think?

This pull request closes #940

### CODE REVIEW

**Comment by KevinAsher** [comment]
@BlackSamsung,  LGTM! 🎉

File: BlackSamsung.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: BlackSamsung
+latitude:25.168
+longitude:75.854
+---
```
Comment (stoe) [inline BlackSamsung.json]: @BlackSamsung, looks good, except for two little missing spaces after `latitude` and `longitude`.

**Comment by hollenberry** [comment]
Hey @BlackSamsung  ! Looks like you've gotten good advice from Stefan above! If you take action on those changes, you'll be set. 👍

Please let us know if you need any help! We'll be closing this out to keep the repository clean if there's no action or conversation within the week.

**Comment by crichID** [comment]
@BlackSamsung I went ahead and fixed the syntax issues in https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/5132/commits/653b20e698b74945ccd44ff044660a1506b59cf7. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by April 21st, I will go ahead and do it just to keep the repository clean!

**Pull request merged**